src/data_store/mock_data_store.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+Object.defineProperty(exports, '__esModule', { value: true });
+const tslib_1 = require('tslib');
+const util = require('../util');
+/**
+ * MockDataStore
+ *
+ * This DataStore is very simple store that save any data in flush memory.
+ */
+class MockDataStore {
```

Comment:
The word Mock is a misnomer because it reminds us of test mocks. `MemoryStore` or similar would be appropriate.